public/examples/download/downloadApp.js

```diff
@@ -33,19 +33,15 @@
     var option;
     var startTestButton;
     var firstRun = true;
-    var downloadSize = 10000;
```

Comment:
did these get removed completely? It seems like there are references to some of these items on line 338